x/model/types/messages_model_test.go

```diff
@@ -283,6 +283,38 @@ func TestMsgCreateModel_ValidateBasic(t *testing.T) {
 			}(validMsgCreateModel()),
 			err: validator.ErrFieldMaxLengthExceeded,
 		},
+		{
+			name: "LsfUrl is not valid URL",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.LsfUrl = "not valid URL"
+				return msg
+			}(validMsgCreateModel()),
+			err: validator.ErrFieldNotValid,
+		},
+		{
+			name: "LsfUrl starts with http:",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.LsfUrl = "http://sampleflowurl.dclmodel"
+				return msg
+			}(validMsgCreateModel()),
+			err: validator.ErrFieldNotValid,
+		},
+		{
+			name: "LsfUrl length > 256",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.LsfUrl = "https://sampleflowurl.dclauth/" + tmrand.Str(257-30) // length = 257
+				return msg
+			}(validMsgCreateModel()),
+			err: validator.ErrFieldMaxLengthExceeded,
+		},
+		{
+			name: "LsfRevision is missing",
+			msg: func(msg *MsgCreateModel) *MsgCreateModel {
+				msg.LsfRevision = 0
+				return msg
+			}(validMsgCreateModel()),
+			err: validator.ErrRequiredFieldMissing,
```

Comment:
It would be great to also add the following (negative) tests:
- LsfRevision < 0
- LsfRevision > 65535